src/DotNet.Status.Web/.config/settings.json

```diff
@@ -136,6 +136,10 @@
     "ApiToken": "[vault(grafana-api-token)]",
     "TableUri": "[vault(deployment-table-sas-uri)]"
   },
+  "Maestro": {
+    "BaseUrl": "https://maestro-prod.westus2.cloudapp.azure.com/",
+    "ApiToken": "[vault(prod-maestro-token)]"
```

Comment:
... prod?  It seems dangerous for something in staging to rely on production, both for security reasons and general rollout stability reasons.  Or is the secret just named weirdly?